src/python/pants/backend/go/util_rules/sdk.py

```diff
@@ -14,14 +15,28 @@
 from pants.engine.env_vars import EnvironmentVars, EnvironmentVarsRequest
 from pants.engine.fs import EMPTY_DIGEST, CreateDigest, Digest, FileContent, MergeDigests
 from pants.engine.internals.selectors import Get, MultiGet
-from pants.engine.process import Process, ProcessResult
+from pants.engine.process import Process, ProcessCacheScope, ProcessResult
 from pants.engine.rules import collect_rules, rule
 from pants.util.frozendict import FrozenDict
 from pants.util.logging import LogLevel
 
+logger = logging.getLogger(__name__)
+
+SANDBOX_CACHE = "gopath"
+NAMED_CACHE = "gopath_cache"
```

Comment:
Maybe add `_PATH` or `_RELPATH` to these constants to make clear that they represent paths?